packages/wonder-blocks-tooltip/components/tooltip-portal-mounter.test.js

```diff
@@ -1,6 +1,7 @@
 // @flow
 import * as React from "react";
-import {mount as enzymeMount} from "enzyme";
+
+import {mount, unmountAll} from "../../../utils/testing/mount.js";
```

Comment:
It's too bad enzyme doesn't automatically `unmountAll` after each test.  Thanks for extracting this an pulling it out to a top level.  At some point we may want to make a separate package in the packages/ folder for test helpers like this.  That way we can do:
```
import {mount, unmountAll} from "@khanacademy/wonder-blocks-test-helpers";
```
and even publish it if we wanted to.

Reply:
Yeah, I thought the same thing about having a wonder blocks testing package.

There's a lengthy issue on enzyme about why they don't have an `unmountAll`. It has to do with them not knowing what test runner is being used and stuff. I think they could still have the helper and then leave it to us to call it, but they don't.